st2api/st2api/controllers/exp/actionalias.py

```diff
@@ -85,7 +85,7 @@ def post(self, action_alias):
 
     @jsexpose(arg_types=[str], body_cls=ActionAliasAPI)
     def put(self, action_alias_id, action_alias):
-        action_alias_db = self.__get_by_id(action_alias_id)
```

Comment:
This also indicates there are no tests for PUT code path - we should work on fixing that.